integrated_channels/sap_success_factors/migrations/0009_sapsuccessfactorsenterprisecustomerconfiguration_deleted_at.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.15 on 2022-10-31 18:55
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sap_success_factors', '0008_auto_20221021_0159'),
+    ]
+
+    operations = [
+        migrations.AddField(
```

Comment:
Does this one not have a historical model?

Reply:
yeah, I'm forgetting why exactly but I do remember talking about that before... I'm pretty sure there's a reason? 😬